app/src/main/java/com/example/sarahtang/memberquiz/OtherActivity.java

```diff
@@ -0,0 +1,196 @@
+        button2 = (Button) findViewById(R.id.button2);
+        button3 = (Button) findViewById(R.id.button3);
+        button4 = (Button) findViewById(R.id.button4);
+        button5 = (Button) findViewById(R.id.button5);
+        endGame = (Button) findViewById(R.id.endGame);
+        memberPic = (ImageView) findViewById(R.id.memberPic);
+        getSupportActionBar().hide();
+
+        memberPic.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent addContactIntent = new Intent(Intent.ACTION_INSERT);
+                addContactIntent.setType(ContactsContract.Contacts.CONTENT_TYPE);
+                addContactIntent.putExtra(ContactsContract.Intents.Insert.NAME,currMember.toString());
+                startActivity(addContactIntent);
+            }
+        });
+
+        final TextView timer = (TextView) findViewById(R.id.timer);
+
+        button2.setOnClickListener(this);
+        button3.setOnClickListener(this);
+        button4.setOnClickListener(this);
+        button5.setOnClickListener(this);
+        endGame.setOnClickListener(this);
+
+        countdowntimer = new CountDownTimer(6000, 1000) {
+
+            public void onTick(long millisUntilFinished) {
+                timer.setText("Time Left: " + millisUntilFinished / 1000);
+            }
+
+            public void onFinish() {
+                nextMember();
+                //timer.setText("done!");
+            }
+        };
+        countdowntimer.start();
+        nextMember();
+
+
+        endGame.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                AlertDialog.Builder builder = new AlertDialog.Builder(OtherActivity.this);
+                builder.setMessage("Are you sure you want to exit?")
+                        .setPositiveButton("OK", new DialogInterface.OnClickListener() {
+                            public void onClick(DialogInterface dialog, int id) {
+                                Intent intent = new Intent(getApplicationContext(), MainActivity.class);
+                                startActivity(intent);
+                            }
+                        })
+                        .setNegativeButton("Cancel", null);
+
+                // Create the AlertDialog object and return it
+                builder.show();
+            }
+        });
+
+    }
+    private void nextMember() {
+        //calling for picture, each turn
+        String[] options = getOptions();
+        button2.setText(options[0]);
+        button3.setText(options[1]);
+        button4.setText(options[2]);
+        button5.setText(options[3]);
+        button2.setBackgroundColor(Color.parseColor("#fdfffc"));
+        button3.setBackgroundColor(Color.parseColor("#fdfffc"));
+        button4.setBackgroundColor(Color.parseColor("#fdfffc"));
+        button5.setBackgroundColor(Color.parseColor("#fdfffc"));
```

Comment:
In this case it would actually be faster to define the color beforehand, won't have to parse it so many times